wasmer/bridge.go

```diff
@@ -21,24 +21,17 @@ type cUint C.uint
 type cUint32T C.uint32_t
 type cUint8T C.uint8_t
 type cWasmerByteArray C.wasmer_byte_array
-type cWasmerExportDescriptorT C.wasmer_export_descriptor_t
```

Comment:
Were these types unused?

Reply:
Yes. I didn't mean to add these changes in this PR, but it seems they slipped in. Just noticed when creating this PR. I can remove them from here, move them to another PR, or do nothing.